flink-runtime/src/main/java/org/apache/flink/runtime/instance/SlotPool.java

```diff
@@ -312,24 +312,36 @@ public void returnAllocatedSlot(Slot slot) {
 		// (1) do we have a slot available already?
 		SlotAndLocality slotFromPool = availableSlots.poll(resources, locationPreferences);
 		if (slotFromPool != null) {
-			SimpleSlot slot = createSimpleSlot(slotFromPool.slot(), slotFromPool.locality());
```

Comment:
Method `createSimpleSlot` is no longer in use.

Reply:
True. Will do it.